internal/tfimport/resources/google_container_cluster.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package resources
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
+)
+
+// GkeClusterImporter defines a struct with the necessary information for a GKE cluster to be imported.
+type GkeClusterImporter struct{}
```

Comment:
GKE

Reply:
Also, optional but if you're only creating importers in this package then it may be more suitable to call this package importer

then do:

var importers = map[string]importer{
	"google_storage_bucket": &importer.StorageBucket{},
	...
}

Reply:
Done, renamed to "GKE" and renamed the package.